packages/client/src/util/rpc.ts

```diff
@@ -243,3 +251,38 @@ export function createWsRPCServerListener(opts: CreateWSServerOpts): jayson.Http
   // Only return something if a new server was created
   return !opts.httpServer ? httpServer : undefined
 }
+
+export /**
```

Comment:
something seems to have went wrong here. the `export` keyword is separated from the `function` keyword by a comment.